cli/mobilecoin/client.py

```diff
@@ -220,8 +220,16 @@ def build_and_submit_transaction(self, account_id, amount, to_address):
                 "addresses_and_values": [(to_address, amount)],
             }
         })
+        return r
+
+    def build_and_submit_transaction(self, account_id, amount, to_address):
+        r = self._build_and_submit_transaction(account_id, amount, to_address)
         return r['transaction_log']
 
+    def build_and_submit_transaction_with_proposal(self, account_id, amount, to_address):
```

Comment:
I think we should just return the entire response. This approach doesn't scale as more data is returned. Suppose we start returning information about the node we submitted the transaction to. Will we now have `build_and_submit_transaction_with_proposal_and_node`?
With that said, I don't feel strongly about it so I am going to approve and defer to @christian-oudard since he wrote most of this.

While we're at the topic of breaking changes, I think if we are starting to treat this client code as "the official full-service python client", we should not be exposing an API that limits the functionality - this API limits to a single outlay, but full-service actually supports multiple outlays.